src/pages/index.js

```diff
@@ -21,38 +25,34 @@ import Section from "../components/Section.js";
 import { PopupWithForm } from "../components/PopupWithForm.js";
 import { PopupWithImage } from "../components/PopupWithImage.js";
 
-const cardsSection = new Section(
-  {
-    data: initialCards,
-    renderer: (item) => {
-      const cardElement = createCard(item);
-      cardsSection.addItem(cardElement);
-    },
-  },
-  cardList
-);
-cardsSection.renderItems();
+import { Api } from "../components/Api.js";
 //
 export const cardPopup = new PopupWithImage(modalCard);
 cardPopup.setEventListeners();
 
-const profileEddited = new UserInfo(profileTitle, profileSubtitle);
+const profileEdited = new UserInfo(profileTitle, profileSubtitle);
```

Comment:
`profileEdited`


It's a `profileInfo` (without `ed`) otherwise it's misleading